docs/howto/operate/new-cluster/index.md

```diff
@@ -0,0 +1,25 @@
+(new-cluster)=
+# Add a new cluster
+
+The first thing we need in deploying a new hub is a Kubernetes Cluster.
+Deploying a new cluster is specific to the cloud provider, though there are many similarities between them all.
+This section covers topics in deploying a new Kubernetes Cluster for the major usecases we serve.
+
+(cluster:when-to-deploy)=
+## When to deploy a new Kubernetes cluster
+
+Deploying a new cluster is a more complex and costly operation, and generally requires more of our time to upkeep and deploy (and thus, will be more costly to the community).
+In general, we'd like hubs to be deployed on pre-existing clusters whenever possible, rather than having a dedicated cluster that is just for the hub's community.
+
+In general, the question of whether to deploy a new cluster for a community comes down to **whether they need to use their own cloud billing accounts**. This is usually either because they have their own cloud credits that they wish to use, or because they work at an institution that mandates (or otherwise incentivizes) individuals to use institutional cloud accounts.
```

Comment:
I would say the other general guidance is the sort of workload they may need, the underlying computational power the community using the infrastructure will have. You can see in [this directory](https://github.com/2i2c-org/pilot-hubs/tree/master/terraform/gcp/projects) different GCP cluster configurations with different sort of computing nodes, for instance.

Reply:
Good point - I've update the section to mention complex infrastructure needs as well